convert.py

```diff
@@ -78,60 +92,60 @@ def generate_schema_block(content):
     block = {
         'type': content.get('block_type'),
         'id': content.get('block_id'),
-        'sections': []
-    }
-
-    section = {
-        'id': content.get('section_id'),
-        'title': content.get('section_title'),
-        'description': content.get('section_description'),
+        'title': content.get('block_title'),
         'questions': []
     }
 
-    if content.get('section_number'):
-        section['number'] = content.get('section_number')
-
-    block['sections'].append(section)
+    question = {
+        'id': content.get('question_id'),
+        'title': content.get('question_title'),
+        'description': content.get('question_description'),
+        'type': 'General',
+        'answers': content.get('answers')
 
-    if content.get('question_id'):
-        question = {
-            'id': content.get('question_id'),
-            'title': content.get('question_title'),
-            'description': content.get('question_description'),
-            'type': 'General',
-            'answers': content.get('answers')
-
-        }
+    }
 
-        if content.get('question_number'):
-            question['number'] = content.get('question_number')
+    if content.get('question_number'):
+        question['number'] = content.get('question_number')
 
-        if content.get('question_guidance'):
-            question['guidance'] = content.get('question_guidance')
+    if content.get('question_guidance'):
+        question['guidance'] = content.get('question_guidance')
 
-        block['sections'][0]['questions'].append(question)
+    block['questions'].append(question)
 
     return block
 
 
 if __name__ == '__main__':
+    pathlib.Path('Blocks').mkdir(parents=True, exist_ok=True)
```

Comment:
Ok, so only create the `Blocks` folder if the user hasn't supplied a `--blocks-out` argument, and if they have it would be useful to create the folder if it doesn't already exist...